syntax = "proto3";

package map;

message PncPathConfig{
  double search_radius = 1;
  double search_theta_diff = 2;
  double look_backward_distance = 3;
  double look_forward_distance = 4;
}
